### PR TITLE
107 Chore: Remove verifyReputation and add readme info

### LINKED ISSUE #107: Chore: Remove verifyReputation task
## User Story
As a dev I want the list of available tasks (`npx hardhat`) to be as short as possible.

## Status Quo
- there is a task `verifyReputation` to verify the Reputation contract on Etherscan
- the task is actually redundant, because there exists CLI command for verifying contracts
- the task unnecessarily blows up the list of available tasks

## Acceptance Criteria
- `verifyReputation` is removed
- the CLI command to verify contracts is explained in the readme

### PR DESCRIPTION
**Pull request by fabianschu**
closes #107

### CODE REVIEW

File: tasks/reputationManagement.js [hunk 1]
```diff
@@ -4,17 +4,6 @@ const initialRepBalances = require("../inputs/initialRepBalances.json");
 
 const { formatEther } = utils;
 
-task("verifyReputation", "Verifies the contract on etherscan").setAction(
```
Comment (Zitzak) [inline tasks/reputationManagement.js:7]: For my own understanding, why did you take this part out but not replacing it with something else? Was this only for testing?

**Review by Zitzak — APPROVED** [review]

File: tasks/reputationManagement.js (hunk 1 above)
Comment (fabianschu) [reply tasks/reputationManagement.js:7]: I believe the information is in the issue :) 
https://github.com/PrimeDAO/contracts-v2/issues/107

Comment (fabianschu) [reply tasks/reputationManagement.js:7]: lmk if it is still unclear

Comment (Zitzak) [reply tasks/reputationManagement.js:7]: You're right, it cleared things up. I didn't think about looking into the issue. Thanks

**Review by codebuster22 — APPROVED** [review]

**Pull request merged**